.azure-pipelines/client.yml

```diff
@@ -0,0 +1,65 @@
+trigger:
+- psSdkJson6
+
+#Variables: 
+#  'source.scope' list of projects to build (Ex: SDKs\KeyVault\dataPlane)
+#  'build.useLatest' indicates whether or not to use build bootstrap files from the LTSBuildTools branch
+
+jobs:
+  - job: 'Build_and_Test'
+  
+    pool:
+      vmImage: 'vs2017-win2016'
+
+    steps:
+      - script: |
+         set _VSWHERE="%ProgramFiles(x86)%\Microsoft Visual Studio\Installer\vswhere.exe"
+         for /f "usebackq tokens=*" %%i in (`%_VSWHERE% -latest -prerelease -property installationPath`) do set _VSCOMNTOOLS=%%i\Common7\Tools
+         echo "%_VSCOMNTOOLS%\VsDevCmd.bat"
+         call "%_VSCOMNTOOLS%\VsDevCmd.bat"
+         
+         sn -Vr *,31bf3856ad364e35
+        displayName: 'Allow unsigned build tool'
+
+      - task: MSBuild@1
+        displayName: 'Init Build'
+        inputs:
+          solution: build.proj
+          msbuildArguments: '/t:Init /flp:LogFile=msbuild.Initlog;Verbosity=normal /p:UseLatest=$(build.useLatest)'
```

Comment:
you know better than I do about devOps pipeline yml.
I noticed you are still passing in UseLatest, but I don't see the variable defined (build.useLatest).
Will it still work?

Reply:
it is defined in the definition.

Reply:
You're not supposed to define variables in the yml.  I had that in an earlier iteration and it results in the variable values you pass in being overwritten.  So I removed them.